config/validator/validation.yaml

```
@@ -23,22 +23,23 @@ App\User\Application\DTO\UserRegisterBatchDto:
-                - NotBlank: { message: 'not.blank' }
-                - Length:
-                    max: 255
-                - App\Shared\Application\Validator\Initials: ~
-              password:
-                - NotBlank: { message: 'not.blank' }
-                - Length:
-                    max: 255
-                - App\Shared\Application\Validator\Password: ~
+              fields:
+                email:
+                  - Email: { message: 'email.invalid' }
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)